### PR TITLE
Implemented heroes trail cards

### PR DESCRIPTION
**Pull request by Lemongrass3110**
* **Addressed Issue(s)**: #1313 

* **Server Mode**: Renewal

* **Description of Pull Request**:

### CODE REVIEW

File: db/re/item_combo_db.txt [hunk 1]
```diff
@@ -679,6 +681,8 @@
 24253:28391,{ bonus2 bAddRace,RC_All,1; bonus bMaxHPrate,1; if (getequiprefinerycnt(EQI_SHADOW_ACC_R)+getequiprefinerycnt(EQI_SHADOW_ACC_L)>=10) { bonus2 bAddRace,RC_All,2; bonus bMaxHPrate,2; } }
 24255:28392,{ bonus2 bAddRace,RC_All,1; bonus bMaxHPrate,1; if (getequiprefinerycnt(EQI_SHADOW_ACC_R)+getequiprefinerycnt(EQI_SHADOW_ACC_L)>=10) { bonus2 bAddRace,RC_All,2; bonus bMaxHPrate,2; } }
 27147:27148,{ bonus bDelayrate,-3; }
+27163:27165,{ bonus bDelayrate,-3; bonus bVit,2; bonus bInt,2; bonus bAgi,2; bonus bDex,2; }
+27166:27167,{ bonus2 bSubEle,Ele_Neutral,5; bonus2 bMagicAddSize,Size_All,10; }
```
Comment (Atemo) [inline db/re/item_combo_db.txt]: `bonus2 bMagicAddSize,Size_All,10;` but the description
`When equipped with Faceworm Larva Card,
increase magical damage against fire property monster by 10%`
?

File: db/re/item_db.txt [hunk 2]
```diff
@@ -11452,6 +11452,34 @@
 27160,Vavayaga_Card,Vavayaga Card,6,20,,10,,,,,,,,64,,,,,{ bonus bFlee,getrefine()*2; },{},{}
 27161,Mavka_Card,Mavka Card,6,20,,10,,,,,,,,136,,,,,{ bonus2 bMagicAtkEle,Ele_Fire,20; bonus2 bMagicAtkEle,Ele_Earth,20; },{},{}
 27162,Gopinich_Card,Gopinich Card,6,20,,10,,,,,,,,769,,,,,{ bonus bSPDrainValue,5; bonus bUseSPrate,50; },{},{}
+27163,Faceworm_Card,Faceworm Card,6,20,,10,,,,,,,,136,,,,,{ bonus bAgi,-1; bonus bDex,-1; bonus bAspdRate,3; },{},{}
+27164,Faceworm_Queen_Card,Faceworm Queen Card,6,20,,10,,,,,,,,64,,,,,{ bonus bMaxHPrate,-10; .@r = getrefine(); bonus bCritical,15 + .@r; bonus bCritAtkRate,.@r; },{},{}
+27165,Dark_Faceworm_Card,Dark Faceworm Card,6,20,,10,,,,,,,,136,,,,,{ bonus bVit,-1; bonus bInt,-1; bonus bDelayrate,-3; },{},{}
+27166,Faceworm_Egg_Card,Faceworm Egg Card,6,20,,10,,,,,,,,64,,,,,{ .@r = getrefine(); .@sub = 2+(.@r >= 9 ? 3 : .@r >= 7 ? 1 : 0); bonus bNearAtkDef,.@sub; bonus bMagicAtkDef,.@sub; },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: From the description `reduce damage taken from fire property monsters`, it shouldn't be `bonus2 bSubDefEle,e,x;    		+x% damage reduction from enemy with defense element e` ?

File: db/re/item_db.txt [hunk 3]
```diff
@@ -11452,6 +11452,34 @@
...
+27167,Faceworm_Larva_Card,Faceworm Larva Card,6,20,,10,,,,,,,,4,,,,,{ bonus2 bSubEle,Ele_Neutral,15; bonus2 bMagicAddSize,Size_All,getrefine()*3; },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: `Increase water property magic damage by 3% per refine rate.` -> `bonus2 bMagicAtkEle,e,x;  		Increases damage of e element magic by x%` ?

File: db/re/item_db.txt [hunk 4]
```diff
@@ -11452,6 +11452,34 @@
...
+27168,Irene_High_Elder_Card,Irene High Elder Card,6,20,,10,,,,,,,,4,,,,,{ bonus bHit,10 + 5 * getrefine()/3; },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: `bonus bHit,10 + 5 * (getrefine()/3);`

File: db/re/item_db.txt [hunk 5]
```diff
@@ -11452,6 +11452,34 @@
...
+27169,Payon_Soldier_Card,Payon Soldier Card,6,20,,10,,,,,,,,2,,,,,{ .@atk = 5; .@r = getrefine(); if(.@r >= 10){ .@atk += 20; if(.@r >= 14){ .@atk += 20; } } } bonus bAtk,.@atk; bonus bMatk,.@atk; },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: `When compounded on spear`
Maybe `getiteminfo( getequipid(EQI_COMPOUND_ON),11 )` to retrieve the view and
```
View:
	4: One-handed spears
	5: Two-handed spears
```

File: db/re/item_db.txt [hunk 6]
```diff
@@ -11452,6 +11452,34 @@
...
+27183,Gigantes_Card,Gigantes Card,6,20,,10,,,,,,,,128,,,,,{ bonus bAtk,20; if(readparam(bStr) >= 120) { bonus bAtk,20; bonus bAspdRate,-3; } },{},{}
+27184,Knight_Sakray_Card,Knight Sakray Card,6,20,,10,,,,,,,,2,,,,,{ bonus bAtk,20; bonus2 bAddRace,RC_Demon,30; bonus2 bAddRace,RC_Undead,30; },{},{}
+27249,Arhi_Card,Arhi Card,6,20,,10,,,,,,,,64,,,,,{ bonus bAtk,15; bonus bMatk,15; bonus bMaxHP,500; bonus bHit,15; bonus bFlee,15; },{},{}
+27250,Dio_Anemos_Card,Dio Anemos Card,6,20,,10,,,,,,,,64,,,,,{ bonus bMatkRate,3; bonus2 bMagicAtkEle,Ele_Neutral,5; bonus2 bMagicAtkEle,Ele_Earth,5; },{},{}
+27251,Geffen_Gang_Member_Card,Geffen Gang Member Card,6,20,,10,,,,,,,,64,,,,,{ bonus bCritAtkRate,3; bonus bAspdRate,5; },{},{}
+27252,Geffen_Bully_Card,Geffen Bully Card,6,20,,10,,,,,,,,64,,,,,{ bonus bCritAtkRate,5; bonus bAspdRate,3; },{},{}
+27253,Geffen_Shoplifter_Card,Geffen Shoplifter Card,6,20,,10,,,,,,,,64,,,,,{ bonus bMatkRate,3; bonus2 bMagicAtkEle,Ele_Fire,5; bonus2 bMagicAtkEle,Ele_Holy,5; },{},{}
+27254,Faymont_Card,Faymont Card,6,20,,10,,,,,,,,64,,,,,{ bonus bSPrecovRate,10; bonus3 bAutoSpellWhenHit,"CH_SOULCOLLECT",1,50; },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: `When receiving damage, has a 1% chance to auto-cast Zen lv. 1.` Why 5% ?

File: db/re/item_db.txt [hunk 7]
```diff
@@ -11452,6 +11452,34 @@
...
+27261,Ju_Card,Ju Card,6,20,,10,,,,,,,,2,,,,,{ .@rate = 10 + (getiteminfo(getequipid(EQI_HAND_R),11) == W_BOOK ? 20 : 0) + (getrefine() >= 14 ? 20 : 0); bonus2 bSkillAtk,"MG_FIREBOLT",.@rate; bonus2 bSkillAtk,"MG_COLDBOLT",.@rate; bonus2 bSkillAtk,"MG_LIGHTNINGBOLT",.@rate; },{},{}
+27262,Dwigh_Card,Dwigh Card,6,20,,10,,,,,,,,136,,,,,{ bonus2 bMagicAtkEle,Ele_Dark,20; bonus2 bMagicAtkEle,Ele_Neutral,20; },{},{}
+27263,Fay_Kanavian_Card,Fay Kanavian Card,6,20,,10,,,,,,,,64,,,,,{ autobonus "{ bonus bDef,300; }",100,10000,BF_WEAPON|BF_SHORT,"{ active_transform 1040,10000; }"; },{},{}
+27264,Evil_Shadow_Card,Evil Shadow Card,6,20,,10,,,,,,,,32,,,,,{ bonus2 bSubEle,Ele_Dark,30; bonus2 bSubEle,Ele_Undead,30; bonus2 bSubEle,Ele_Holy,-30; bonus2 bSubEle,Ele_Fire,-30; },{},{}
+27265,Evil_Fanatics_Card,Evil Fanatics Card,6,20,,10,,,,,,,,2,,,,,{ bonus2 bAddEle,Ele_Dark,30; bonus2 bAddEle,Ele_Undead,30; bonus2 bMagicAddEle,Ele_Dark,30; bonus2 bMagicAddEle,Ele_Undead,30; bonus2 bAddEle,Ele_Holy,-30; bonus2 bAddEle,Ele_Fire,-30; bonus2 bMagicAddEle,Ele_Holy,3-0; bonus2 bMagicAddEle,Ele_Fire,-30; },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: Typo `bonus2 bMagicAddEle,Ele_Holy,3-0;`

File: db/re/item_combo_db.txt (hunk 1 above)
Comment (secretdataz) [reply db/re/item_combo_db.txt]: From kRO's itemInfo.lub
`화속성 적에게 주는 마법 데미지 10% 증가`
Translate to Increases magical damage by 10% (may be mistranslated)

File: db/re/item_db.txt (hunk 4 above)
Comment (secretdataz) [reply db/re/item_db.txt]: getrefine()/3 is already highest precedence.

File: db/re/item_combo_db.txt (hunk 1 above)
Comment (Lemongrass3110) [reply db/re/item_combo_db.txt]: Yeah if not blame it on me (in hash: d634cf6).
I saw iRO did not match kRO and thought we better go for kRO.

Comment (Atemo) [reply db/re/item_combo_db.txt]: Ha alright. For infos I was referring to
https://www.divine-pride.net/database/item/27166
https://www.divine-pride.net/forum/index.php?/topic/2788-kro-changelog-september-13-2017/&tab=comments#comment-3820

**Pull request merged**